calypso/csadmin/clicontracts/write.go

```diff
@@ -230,6 +230,19 @@ func WriteGet(c *cli.Context) error {
 		return errors.New("didn't get a write instance: " + err.Error())
 	}
 
+	if c.Bool("export") {
+		_, buf, _, _, err := proof.KeyValue()
+		if err != nil {
+			return errors.New("failed to get value from proof: " + err.Error())
```

Comment:
Can we use `xerrors.Errorf("...: %v", err)` here please to be consistent with #2116